input/pagecontent/reference.md

```diff
@@ -2114,6 +2197,42 @@ Invariants are defined using the keywords `Invariant`, `Description`, `Expressio
   XPath:      "f:given or f:family"
   ```
 
+#### Defining Logical Models
+
+Logical models allow authors to define new structures representing arbitrary content. While profiles can only add new properties as formal extensions, logical models can add properties as standard elements with standard paths. Logical models have many uses, [as described in the FHIR specification](http://hl7.org/fhir/R4/structuredefinition.html#logical), but are often used to convey domain-specific concepts in a user-friendly manner. Authors often use logical models as a basis for defining formal profiles in FHIR.
+
+Logical models are defined in FSH using the keyword `Logical`. The keywords `Parent`, `Id`, `Title`, and `Description` are OPTIONAL.
+
+If no `Parent` is specified, the empty [Base](http://hl7.org/fhir/2021May/types.html#Base) type is used as the default parent. Note that the Base type does not exist in FHIR R4, but both SUSHI and the FHIR IG Publisher have implemented special case logic to support Base in FHIR R4. Authors who wish to have top-level id and extension elements should use [Element](http://hl7.org/fhir/R4/element.html) as the logical model's parent instead. Authors may also specify another logical model, a resource, or a complex datatype as a logical model's parent.
+
+Rules defining the logical model follow immediately after the keyword section. Logical models are primarily comprised of AddElement rules. Since logical models are represented using StructureDefinition, many of the rules used in profile definitions may also be used when defining logical models. According to the FHIR specification's [interpretation of ElementDefinition in different contexts](http://hl7.org/fhir/R4/elementdefinition.html#interpretation), however, logical models may not use slicing or fixed/patterned values.  As a result, Contains rules and Assignment rules are forbidden in logical model definitions. Logical models are also prohibited from constraining elements inherited from a parent definition.
+
+**Example:**
+
+* Define a logical model for a human and their family members:
+
+  ```
+  Logical:        Human
+  Title:          "Human Being"
+  Description:    "A member of the Homo sapien species."
+  * name 0..* SU HumanName "Name(s) of the human" "The names by which the human is or has been known"
+  * birthDate 0..1 SU dateTime "The date of birth, if known"
+      "The date on which the person was born. Approximations may be used if exact date is unknown."
+  * deceased[x] 0..1 SU boolean or dateTime or Age "Indication if the human is deceased"
+      "An indication if the human has died. Boolean should not be used if date or age at death are known."
+  * family BackboneElement 0..1 "Family" "Members of the human's immediate family."
```

Comment:
Looks like `BackboneElement` and `0..1` are out of order.

Reply:
I knew I probably should have tried compiling the examples before documenting them.  But I thought "Nah, they're so simple.  What could you possibly screw up?"

Reply:
@ngfreiter -- fixed (along w/ a couple of other things) in daeda35.